src/main/resources/api/schemas/legacy/openingDayInfo.yaml

```diff
@@ -0,0 +1,39 @@
+description: An entire day's opening time
```

Comment:
Given that this is a total replacement of the calendar API, what is the purpose of keeping around the _legacy_ schemas in the documentation?

Reply:
> @marcjohnson-kint As part of the [upgrade script](https://github.com/folio-org/mod-calendar/blob/ualibweb/new/src/main/java/org/folio/calendar/service/CustomTenantService.java#L92), the existing RMB data is mapped into the legacy `Period` and `OpeningDay` types before being transformed into the new spec.

An alternative to keeping the schema around and continuing to generate classes from it could have been to move to non-generated classes (given that they were no longer relevant for the API definitions and weren't expected to change in the future).

Reply:
@marcjohnson-kint Would that be preferred?

Reply:
It's a trade off I would personally make. The classes could be isolated to within the migration undertaken during the upgrade and the schema files, which are effectively noise, could be removed from the API documentation.

(I am biased, I am not a fan in general of code generation from API documentation and I'm even less keen in putting extra stuff in there as a convenient way to trigger it)

Reply:
Implemented the change